apps/server/src/modules/file-storage-client/interfaces/file-domain-object-props.ts

```diff
@@ -0,0 +1,10 @@
+import { EntityId } from '@shared/domain';
+import { FileRecordParamsParentTypeEnum } from '../fileStorageApi/v3';
+
+export interface IFileDomainObjectProps {
+	id: EntityId;
+	name: string;
+	parentType: FileRecordParamsParentTypeEnum;
```

Comment:
Like mentioned in another file: Shouldn't we use `FileRecordParentType` from filerecord.entity.ts here?